tests/electron/helpers/global-setup.js

```diff
@@ -39,6 +40,7 @@ exports.stopApplication = async () => {
 	global.electronApp = null;
 	global.page = null;
 	process.env.MOCK_DATE = undefined;
+	process.env.jstest = undefined;
```

Comment:
why is it now a "jstest" instead of "intest" ?

Reply:
weird, wonder where that came from. will fix